### ISSUE TITLE
[内网测试]

### ISSUE THREAD

**Issue by guihousun** [guihousun/Competition_HW#6]
### 被测提交 SHA

？

### 官方平台与规则版本

？

### 测试结果

需求或规则澄清

### 最小复现与实际结果

？

### 预期行为与官方依据

？

### 相关 Issue / PR 与补充信息

3、为啥CoreGeek.tar.gz压缩包解压失败了？

打包tar.gz包有2种方式：第一种用tar指令；第二种用7zip来两次压缩，不能用winRAR进行压缩，winRAR只能压缩成rar或者zip形式：

4、python3.11到底有哪些库给我用？有没有清单？
详见群空间

absl-py==2.1.0
astunparse==1.6.3
blinker==1.8.2
cachetools==5.5.0
certifi==2024.8.30
charset-normalizer==3.3.2
click==8.1.7
filelock==3.16.0
Flask==3.0.3
flatbuffers==24.3.25
fsspec==2024.9.0
gast==0.6.0
google-auth==2.34.0
google-pasta==0.2.0
grpcio==1.66.1
h5py==3.11.0
idna==3.8
itsdangerous==2.2.0
Jinja2==3.1.4
keras==3.5.0
Keras-Preprocessing==1.1.2
libclang==18.1.1
Markdown==3.7
markdown-it-py==3.0.0
MarkupSafe==2.1.5
mdurl==0.1.2
ml-dtypes==0.4.0
mpmath==1.3.0
namex==0.0.8
networkx==3.3
numpy==1.26.4
opencv-python==4.10.0.84
opt-einsum==3.3.0
optree==0.12.1
packaging==24.1
pandas==2.2.2
pillow==10.4.0
protobuf==4.25.4
pyasn1==0.6.0
pyasn1_modules==0.4.0
Pygments==2.18.0
python-dateutil==2.9.0.post0
pytz==2024.1
PyYAML==6.0.2
requests==2.32.3
rich==13.8.0
rsa==4.9
six==1.16.0
sympy==1.13.2
tensorboard==2.17.1
tensorboard-data-server==0.7.2
tensorflow==2.17.0
tensorflow-io-gcs-filesystem==0.37.1
termcolor==2.4.0
torch==2.4.1
torchvision==0.19.1
triton==3.0.0
typing_extensions==4.12.2
tzdata==2024.1
urllib3==2.2.2
Werkzeug==3.0.4
wrapt==1.16.0

### 数据范围

- [x] 我确认以上信息已脱敏并允许公开，附件不含公司凭据或未获允许的数据。

**Comment by guihousun** [comment]
已收到，按“官方资料补充”处理，不作为某次代码运行失败。

这份 FAQ 补齐了全局台账 V-009 的一部分：Python 3.11 及 62 项库版本，例如 numpy 1.26.4、pandas 2.2.2、requests 2.32.3、PyYAML 6.0.2、torch 2.4.1、tensorflow 2.17.0。来源登记为你转贴的官方 FAQ；尚不能据此确认操作系统、CPU/内存限额、沙盒目录/权限、跨回合文件持久化，以及参赛进程与 executeCmd 沙盒是否使用同一环境。

打包要求也收到：使用真正的 tar + gzip 格式，不能仅把 ZIP/RAR 改名为 .tar.gz。本地对仓库原始 Demo/CoreGeek.tar.gz 做了 gzip/tar 完整读取检查，10 个成员均可读取，未复现解压失败；这只是本地包完整性验证，不是官方平台验收。该压缩包仍是官方原始示例，不包含我们当前模拟器和策略的全部改动。

另外，“被测提交 SHA”就是测试代码的版本号。当前完整项目版本为 aa6eb3e79c96e15e40ebc0c58817dd031ac6c933；这类单纯补充文档的 Issue 可填写“不适用（资料补充）”，无需猜测 SHA。

**Comment by guihousun** [comment]
“被测提交 SHA”就是测试代码的版本号——在哪看呀？

**Comment by guihousun** [comment]
在公司电脑的项目文件夹打开终端，执行：

```powershell
git rev-parse HEAD
git status --short
```

第一条输出的完整 40 位字符就是“被测提交 SHA”，直接复制进 Issue。第二条若有输出，说明本地还有改动，也请一起说明；只有 SHA 不能代表这些未提交改动。

也可以在 GitHub 仓库先切到你实际使用的分支，点击文件列表上方的最新提交记录，再复制完整 SHA。网页显示的是远程版本，以公司电脑执行第一条的结果为准。

目前完整模拟器项目在 `codex/sgh`，本次发布版本为 `aa6eb3e79c96e15e40ebc0c58817dd031ac6c933`；不要默认认为 main 已包含这次完整发布。若只是下载了 ZIP、目录没有 .git，请记录下载所对应的提交链接；仅凭目录名无法确定版本。

你这条 Issue 是资料补充，SHA 填“不适用”即可。
